lib/lita/handlers/jenkins.rb

```diff
@@ -1,6 +1,13 @@
 require 'json'
 require 'base64'
 
+# Determine if a given String is contains only a number.
+class String
+  def is_i?
```

Comment:
The `is` in the method name is redundant with the `?` so I'd like to remove that...but naming it `i?` would be confusing. Maybe a better name would be `number?`.

If you look at rails they don't use `is` with `?`. For example, `can_pass?`, `new?`, `accepted?`, etc.